packages/runtime/src/lib/stateful.ts

```diff
@@ -3,6 +3,24 @@ import { modelsv2 } from "algosdk";
 import { RUNTIME_ERRORS } from "../errors/errors-list";
 import { RuntimeError } from "../errors/runtime-errors";
 import { StackElem } from "../types";
+import { MAX_KEY_BYTES, MAX_KEY_VAL_BYTES } from "./constants";
+
+/**
+ * Allow the combination of key-value pairs to consume 128 bytes.
```

Comment:
what do you mean by consume here ? could you please add more explanation here ?

Reply:
done